### PR TITLE
Docsite feat

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description : 

PR contains changes for the following components:

- [Modal](https://github.com/humzahasan/FeedyUI/pull/31/commits/1a32f46ed05ccfb0ece2d8f0742000cb2dd1cd77)
- [Sub Menu](https://github.com/humzahasan/FeedyUI/pull/31/commits/09f01f4a9d62b729611cbd3f8c0f2c669dedad6b)
- [Input Slider](https://github.com/humzahasan/FeedyUI/pull/31/commits/616a1ced1fbebc85237798929a8f9dc714c322e9)

[Preview Link](https://deploy-preview-31--feedyui.netlify.app/documentation/documentation.html)

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! UI looks great!

Good to merge 👍🏻

File: documentation/documentation.html [hunk 1]
```diff
@@ -177,5 +188,22 @@
...
+          console.log(navMenu);
+          navMenu.classList.toggle("navbar-list-res");
+        })
+      );
+      document.querySelectorAll(".nav-menu-mobile").forEach((item) => {
+        item.addEventListener("click", () => {
+          console.log("I am cliked", item);
+          let navMenu = document.querySelector(".navbar-items");
+          console.log(navMenu);
+          navMenu.classList.toggle("navbar-list-res");
```
Comment (sudipkundu999) [inline documentation/documentation.html]: Remove console logs

File: library/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  width: 1.25rem;
+}
+
+.myclose {
+  fill: white;
+}
+
+.modal-content {
+  width: 50%;
+  height: auto;
```
Comment (sudipkundu999) [inline library/modal/modal.css]: height is auto by default

File: documentation/doc.css [hunk 3]
```diff
@@ -20,6 +20,16 @@
   border-radius: 5px;
 }
 
+.modal-container {
+  padding: 0;
+  height: 800px;
```
Comment (sudipkundu999) [inline documentation/doc.css]: Suggestion to use rem here

**Review by sudipkundu999 — APPROVED** [review]
Clean code. Awesome UI.
Good to merge with changes.

File: library/modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  margin: auto;
+
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  /* z-index: 999; */
+}
```
Comment (bhushannaikaroskar) [inline library/modal/modal.css]: remove commented code

File: documentation/documentation.html [hunk 5]
```diff
@@ -177,5 +188,22 @@
...
+        item.addEventListener("click", () => {
+          let navMenu = document.querySelector(".navbar-items");
+          console.log(navMenu);
+          navMenu.classList.toggle("navbar-list-res");
+        })
+      );
+      document.querySelectorAll(".nav-menu-mobile").forEach((item) => {
+        item.addEventListener("click", () => {
+          console.log("I am cliked", item);
+          let navMenu = document.querySelector(".navbar-items");
```
Comment (bhushannaikaroskar) [inline documentation/documentation.html]: If possible please remove console logs. You can add them again while adding improvements

File: library/card/card.css [hunk 6]
```diff
@@ -15,7 +16,7 @@
 }
 
 .card:hover {
-  box-shadow: 3px 2px 5px #fc7f194b;
+  box-shadow: 3px 2px 5px #857365c5;
 }
```
Comment (bhushannaikaroskar) [inline library/card/card.css]: you can use variables for color

**Review by bhushannaikaroskar — APPROVED** [review]
Hey humza you code is short and easily readable good work. I have given some suggestion do reflect on it before merging. Approving the merge

File: documentation/documentation.html [hunk 7]
```diff
@@ -177,5 +188,22 @@
           obj.contentWindow.document.documentElement.scrollHeight + "px";
       }
     </script>
+    <script>
+      document.querySelectorAll("#toggleBtn").forEach((item) =>
+        item.addEventListener("click", () => {
+          let navMenu = document.querySelector(".navbar-items");
+          console.log(navMenu);
```
Comment (zoyatanweer) [inline documentation/documentation.html]: remove console.log

File: library/modal/modal.css [hunk 8]
```diff
@@ -0,0 +1,47 @@
+.modal {
+  display: none;
+  background-color: #00000034;
```
Comment (zoyatanweer) [inline library/modal/modal.css:3]: use variable for color

**Review by zoyatanweer — APPROVED** [review]
- code is clean and readable
- great work
- good to merge

File: documentation/documentation.html (hunk 1 above)
Comment (humzahasan) [reply documentation/documentation.html]: Done

File: library/modal/modal.css (hunk 4 above)
Comment (humzahasan) [reply library/modal/modal.css]: done

File: documentation/documentation.html (hunk 5 above)
Comment (humzahasan) [reply documentation/documentation.html]: done

File: documentation/documentation.html (hunk 7 above)
Comment (humzahasan) [reply documentation/documentation.html]: done

File: documentation/doc.css (hunk 3 above)
Comment (humzahasan) [reply documentation/doc.css]: done

File: library/modal/modal.css (hunk 2 above)
Comment (humzahasan) [reply library/modal/modal.css]: Okay, will remove

File: library/card/card.css (hunk 6 above)
Comment (humzahasan) [reply library/card/card.css]: Done

File: library/modal/modal.css (hunk 8 above)
Comment (humzahasan) [reply library/modal/modal.css:3]: Okay sure, thanks for pointing this out!

**Pull request merged**